stats/sexbound/sexbounddefeat.lua

```diff
@@ -0,0 +1,568 @@
+	if self:isMonster() and not self._config.enableStartDialogForMonsters then return end
+	local species = self:findSpecies()
+	local defeatedDialog = self._config.startDialog.defeated[species] or
+	self._config.startDialog.defeated.default
+	world.sendEntityMessage(
+		self._entityId, "Sexbound:Actor:Say",
+		{
+			message = util.randomChoice(defeatedDialog)
+		}
+	)
+end
+
+function SexboundDefeat:outputVictoryDialog()
+	if self._hostileEntityType == "npc" and not self._config.enableStartDialogForNPCs then return end
+	if self._hostileEntityType == "monster" and not self._config.enableStartDialogForMonsters then return end
+	local controllerSpecies = world.entitySpecies(self._hostileEntityId)
+	local victoryDialog = self._config.startDialog.victory[controllerSpecies] or
+	self._config.startDialog.victory.default
+	world.sendEntityMessage(
+		self._hostileEntityId, "Sexbound:Actor:Say",
+		{
+			message = util.randomChoice(victoryDialog)
+		}
+	)
+end
+
+function SexboundDefeat:retrieveSexboundConfig(successCallback, failureCallback, entityId)
+	self._promises:add(
+		world.sendEntityMessage(
+			entityId,
+			"Sexbound:Config:Retrieve"
+		),
+		successCallback,
+		failureCallback
+	)
+end
+
+function SexboundDefeat:handleRetrieveSexboundConfigSuccess(sexboundConfig, hostileEntityId)
+	table.insert(self._hostileEntities, 1, {hostileEntityId, sexboundConfig})
+
+	if #self._hostileEntities > 8 then
+		table.remove(self._hostileEntities)
+	end
+	--[[ Output the victory dialog
+	self:outputVictoryDialog()
+	self:transform(sexboundConfig, function(result)
+		result = result or {}
+		if result.uniqueId == nil then
+			return self:handleTransformFailed()
+		end
+		return self:handleTransformSuccess(result.uniqueId)
+	end, function()
+		return self:handleTransformFailed()
+	end,
+	self._actorData
+	)
+	]]
+end
+
+
+function SexboundDefeat:transform(sexboundConfig, successCallback, failureCallback, actorData)
+	local position = nil
+	local entityId = self._entityId
+	if self:isPlayer() then
+		position = entity.position()
+		entityId = self._hostileEntityId
+	end
+	self._promises:add(
+		world.sendEntityMessage(
+			entityId,
+			"Sexbound:Transform",
+			{
+				responseRequired = true,
+				sexboundConfig = sexboundConfig,
+				timeout = 500,
+				position = position
+			},
+			actorData or nil
+		),
+		successCallback,
+		failureCallback
+	)
+end
+
+function SexboundDefeat:handleTransformFailed()
+	self:tryToDie("Transform failed")
+	return false
+end
+
+function SexboundDefeat:handleTransformSuccess(nodeUniqueId)
+	self._sexNodeId = nodeUniqueId
+	self:setIsTransformed(true)
+	self:setTimer(0)
+	self:outputDefeatedDialog()
+	self:_initTimeout()
+	if self:isPlayer() then
+		-- Check config for using invisible "UI" that allows escape with ESC key
+		if self._config.enableDefeatedPlayerEscape then world.sendEntityMessage(self._entityId, "Sexbound:UI:Show", {config = self:loadUIConfig()}) end
+		status.addEphemeralEffect("dontstarve",    self._timeout)
```

Comment:
I have removed the starve protection from all sexnodes so far, so for consistency this needs to go aswell.

**HOWEVER**, this brought up another question. What happens if you die inside of a sexnode? E.g. starve to death mid sex. How does defeat handle this?

Reply:
You can still take damage while in sexnodes, since being defeated is the only thing that prevents damage at the moment. If you die, something weird happens and not only are you killed, but then you also die. Which is a funny way to say you die twice.

Anyway I can remove that.